compiler/noirc_frontend/src/ast/statement.rs

```diff
@@ -770,13 +771,60 @@ impl LValue {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct ForBounds(
+    /*start:*/ pub Expression,
+    /*end:*/ pub Expression,
+    /*inclusive:*/ pub bool,
```

Comment:
What do you think about making these named fields?

Reply:
I think now that we have constructor methods for them it would be a good change; where they are used in pattern matching the syntax would be very similar, just `{ start, end, inclusive }` instead of `(from, to, inclusive)` 👍

Reply:
Renamed to `test_single_multi_char`.